scripts/sync_codemirror.sh

```diff
@@ -101,7 +108,10 @@ process_repo() {
   cd "${tmp_dir}/${org_repo}" || return 1
   git checkout -b "${branch}" || return 1
 
-  git rm -r .
+  if [[ -z "${should_clear}" ]]; then
```

Comment:
```suggestion
  if [[ -n "${should_clear}" ]]; then
```

Reply:
We probably want to check for "true" excplicitly

Reply:
Why wouldn't it be required to delete files?

Reply:
it's in case you just want to copy the source code and not the way to manage the package (a.k.a the fine `package.json` for example).

As you can see in the prometheus source, you don't have the direct way to build the lezer grammar nor the dependencies required, they are at the codemirror-promql package level.

So for this particular sync, copying just the source is simpler (at least from my point of view)